superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -164,11 +164,12 @@ class HeaderActionsDropdown extends React.PureComponent {
         break;
       }
       case MENU_KEYS.TOGGLE_FULLSCREEN: {
+        const hasStandalone = !!getUrlParam(URL_PARAMS.standalone);
         const url = getDashboardUrl(
           window.location.pathname,
           getActiveFilters(),
           window.location.hash,
-          getUrlParam(URL_PARAMS.standalone),
+          !hasStandalone,
```

Comment:
I suppose it would also be correct to do
```js
!getUrlParam(URL_PARAMS.standalone)
```
here?

Reply:
right, but I am surprised to do "negation" after returning a number type or null type.